lessons/module-3/react-2-the-how.md

```diff
@@ -1068,8 +981,8 @@ Take a few minutes to journal:
 * What "aha" moments did you have?
 * Which concepts are the fuzziest for you right now?
 * Which concepts are the clearest for you right now?
-* What do you know about class-based components?
 * What do you know about function-based components?
+* What do you know about hooks?
```

Comment:
Love the checks for understanding.  Potentially could use `<section class="checks-for-understanding">` to group these up.